packages/debug/src/browser/debug-session.tsx

```diff
@@ -368,13 +368,19 @@ export class DebugSession implements CompositeTreeElement {
     }
 
     protected async runInTerminal({ arguments: { title, cwd, args, env } }: DebugProtocol.RunInTerminalRequest): Promise<DebugProtocol.RunInTerminalResponse['body']> {
-        const terminal = await this.doCreateTerminal({ title, cwd, env });
+        const terminal = await this.doCreateTerminal({ title, cwd, env, useServerTitle: false });
```

Comment:
Where `useServerTitle` is used?

Reply:
@akosyakov 
It's from `TerminalWidgetOptions. Please refer to:
https://github.com/eclipse-theia/theia/blob/938bc4885bc0257f3d2d467d6933f102734961ef/packages/terminal/src/browser/base/terminal-widget.ts#L89-L93

Setting it to `false` here makes the title of the integrated in accordance with `RunInTerminalRequestArguments.title` (please refer to propose 2 in #6767).